package.nls.pt-br.json

```diff
@@ -66,6 +67,11 @@
   "configuration.SAS.log.showOnExecutionStart": "Mostrar log SAS quando a execução começa",
   "configuration.SAS.notebook.export.includeLog": "Incluir o log SAS no notebook exportado",
   "configuration.SAS.problems.log.enabled": "Mostre os problemas do log do SAS",
+  "configuration.SAS.results.html.custom.style": "Especifica qual estilo SAS usar para os temas claro, escuro, alto contraste ou alto contraste claro. Esta opção é aplicada quando \"\"#SAS.results.html.style#\"\" está definido como \"\"(auto)\"\".",
```

Comment:
This is NOT correct. "`" should be preserved for VS Code to link to the setting.